alacritty_terminal/src/term/mod.rs

```diff
@@ -887,6 +1018,11 @@ impl<T> Term<T> {
             clipboard,
             event_proxy,
             is_focused: true,
+            last_highlight: None,
+            last_selection: None,
+            damage: DamageRect { x: 0, y: 0, end_x: 0, end_y: 0 },
```

Comment:
What do you think about adding impl Default (you can just derive it) for DamageRect?